### ISSUE TITLE
[tradfri] Color bulbs incorrectly discovered as color temperature bulbs

### ISSUE THREAD

**Issue by ngrodum** [openhab/openhab-addons#6258]
## Expected Behavior
Add a color bulb to the Ikea bridge, look in the "Inbox" in paper-ui, expect a color bulb (with thing type tradfri:0210)

## Current Behavior
Add a color bulb to the Ikea bridge, look in the "Inbox" in paper-ui, find a color temperature bulb (with thing type tradfri:0220)

## Possible Solution
I have just peeked at the source code, and see the binding seems to look for "CWS" in the display name during discovery. From looking at the debug logs, it might appear that Ikea has changed the description of the bulbs from CWS to C/WS.

## Your Environment
* Version used: Openhab 2.4, latest 2.5 snapshot of tradfri binding.

**Comment by ngrodum** [comment]
2019-10-20 23:02:17.370 [DEBUG] [ternal.handler.TradfriGatewayHandler] - requestDeviceDetails response: 
{
   "3311":[
      {
         "5850":1,
         "5709":32886,
         "5851":216,
         "5707":5309,
         "5708":52400,
         "5710":27217,
         "5706":"efd275",
         "9003":0
      }
   ],
   "9001":"Mushroom lamp",
   "9002":1571036916,
   "9020":1571588312,
   "9003":65539,
   "9054":0,
   "9019":1,
   "3":{
      "0":"IKEA of Sweden",
      "1":"TRADFRI bulb E27 C/WS opal 600",
      "2":"",
      "3":"1.3.009",
      "6":1
   },
   "5750":2
}

**Comment by ngrodum** [comment]
I looked at:
https://github.com/openhab/openhab2-addons/blob/master/bundles/org.openhab.binding.tradfri/src/main/java/org/openhab/binding/tradfri/internal/discovery/TradfriDiscoveryService.java#L111

```
if (model != null && model.contains(COLOR_MODELS_IDENTIFIER)) {
    thingType = THING_TYPE_COLOR_LIGHT;
}
```
and:
`private static final String COLOR_MODELS_IDENTIFIER = "CWS";`

**Comment by chriscolden** [comment]
This is odd. I have one of those bulbs i bought at the weekend. Firmware is the same version but mine shows as "TRADFRI bulb E27 CWS opal 600lm" without the slash. openHAB did corretly identify this as a color light. It might be that the older firmware which this bulb would have had out the box identified as TRADFRI bulb E27 CWS opal 600lm which has later been changed to TRADFRI bulb E27 C/WS opal 600" and it's since been updated.

**Comment by cweitkamp** [comment]
Closed by #6506
